### PR TITLE
Update allocated-altitudes.md

### PR DESCRIPTION
**Pull request by asavioliMSFT**
I have this filtered driver bfs on my laptop, but didn't find any info about it on the link

image

### CODE REVIEW

File: windows-driver-docs-pr/ifs/allocated-altitudes.md [hunk 1]
```diff
@@ -1601,6 +1601,12 @@ To see how a driver uses its altitude number in its INF file, see [Creating an I
 | wimfsf.sys | 161000 | Microsoft |
 | GEFCMP.sys | 160100 | Symantec |
 
+## 150000 - 159999: FSFilter Encryption
```
Comment (PiJoCoder) [inline windows-driver-docs-pr/ifs/allocated-altitudes.md:1604]:
```suggestion
## 150000 - 159999: FSFilter Brokering File system
```

**Comment by asavioli** [comment]
Please confirm if the category for this driver we suggested is the proper one

## 150000 - 159999: **FSFilter Brokering File system**

**Review by lorihollasch — COMMENTED** [review]
Sorry, @asavioli, but I cannot accept a PR to manually update this table. I run a script on the internal database of allocated altitudes, and I'm not seeing bfs.sys|150000 in the database. (I'm not seeing anything from 150000-159999.)

**Comment by PiJoCoder** [comment]
@lorihollasch thanks. We started a separate thread with the development team - added you on it. Apparently the team that owns the driver didn't request an Altitude for it.

**Comment by lorihollasch** [comment]
Thanks for starting email thread, @PiJoCoder. I'm going to close out this PR. Once bfs has officially received the allocation (entered in the database), I'll rerun my script to update the table. Will watch email for notification!

**Pull request closed without merging**